the-collab-lab/tcl-54-smart-shopping-list
# Issue 24 | ReadMe file sections and fix rendering error

## Description


Our previous Read Me file that was created by TCL has been renamed `PROJECT-SETUP.md` and is only there for reference. `README.md` has been updated with details about our specific project.

## Related Issue

Closes #54 

## Acceptance Criteria
### ReadMe sections:

- [x] Title + description
- [x] Why We built this: purpose & who it's for
- [x] Technologies used
- [x] Live Demo (github pages link)
- [x] Features ("User can...", etc.)
- [x] Stretch Features (if applicable)
- [x] Preview (GIF or screenshots of UI)
- [x] Testing/QA + System Requirements

### Other
- [x] Center align the home page content and add padding before the title

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|   ✓  | :bug: Bug fix              |
|   ✓  | :scroll: Docs              |

## Testing Steps / QA Criteria

1. From your terminal, first run git pull on main and then git checkout `readme-file-and-other-error-handling`
2. **Important:** instead of npm start, do this: `npm run build` then `npm run preview`. This will open http://localhost:4173/, which will show our app _in production_. The loading error we were previously getting only occurs during production not development! So it's important to check our app in this manner.
   - **Note:** every time you change something in the code and want to check the updates in production, you always have to `npm run build` to update the build, then `npm run preview` again. The most current version in production is always based on the last time you ran the build.
 
3. You can preview what the README file will look like on Github while editing it in VSCode using the command `Ctrl+V K` when the file is open:

![Screenshot 2023-03-06 143931](https://user-images.githubusercontent.com/102492480/223267782-21cc2c13-11a9-4e33-9c2e-cb43bd66b4c4.jpg)
